test/workspace_test.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'test_helper'
+
+describe "Workspace class" do
+
+  it "returns a list of users correctly" do
+    VCR.use_cassette("users_list") do
+      response = SlackCLI::Workspace.new
+      expect(response.users_list).must_be_kind_of Array
+    end
+  end
```

Comment:
Similar to the comment I left in the `channel_test` file...

`response` as a variable name here is understandable, since we get a response after our API call that happens in `SlackCLI::Workspace.new`. However, what `SlackCLI::Workspace.new` ultimately does is create an instance of workspace, and in this test, the variable name could reflect that.

Consider:

```suggestion
  it "returns a list of users correctly" do
    VCR.use_cassette("users_list") do
      workspace = SlackCLI::Workspace.new
      expect(workspace.users_list).must_be_kind_of Array
    end
  end
```